doc/releases/changelog-0.12.0.md

```diff
@@ -380,95 +452,96 @@
+  for large quantum programs.
   [(#1774)](https://github.com/PennyLaneAI/catalyst/pull/1774)
   [(#1828)](https://github.com/PennyLaneAI/catalyst/pull/1828)
 
-* PennyLane's arbitrary-basis measurement operations, such as
-  :func:`qml.ftqc.measure_arbitrary_basis() <pennylane.ftqc.measure_arbitrary_basis>`, are now
-  QJIT-compatible with program capture enabled.
+* PennyLane's arbitrary-basis measurement operations, such as 
+  :func:`qml.ftqc.measure_arbitrary_basis <pennylane.ftqc.measure_arbitrary_basis>`, are now 
+  qjit-compatible with PennyLane program capture enabled.
   [(#1645)](https://github.com/PennyLaneAI/catalyst/pull/1645)
   [(#1710)](https://github.com/PennyLaneAI/catalyst/pull/1710)
 
-* The utility function `EnsureFunctionDeclaration` is refactored into the `Utils` of the `Catalyst`
-  dialect, instead of being duplicated in each individual dialect.
+* The utility function `EnsureFunctionDeclaration` has been refactored into the `Utils` of the 
+  Catalyst dialect instead of being duplicated in each individual dialect.
   [(#1683)](https://github.com/PennyLaneAI/catalyst/pull/1683)
 
-* The assembly format for some MLIR operations now includes adjoint.
+* The assembly format for some MLIR operations now includes `adjoint`.
   [(#1695)](https://github.com/PennyLaneAI/catalyst/pull/1695)
 
-* Improved the definition of `YieldOp` in the quantum dialect by removing `AnyTypeOf`
+* Improved the definition of `YieldOp` in the quantum dialect by removing `AnyTypeOf`.
   [(#1696)](https://github.com/PennyLaneAI/catalyst/pull/1696)
 
-* The assembly format of `MeasureOp` in the `Quantum` dialect and `MeasureInBasisOp` in the `MBQC` dialect now contains the `postselect` attribute.
+* The assembly format of `MeasureOp` in the `Quantum` dialect and `MeasureInBasisOp` in the `MBQC` 
+  dialect now contains the `postselect` attribute.
   [(#1732)](https://github.com/PennyLaneAI/catalyst/pull/1732)
 
-* The bufferization of custom catalyst dialects has been migrated to the new one-shot
-  bufferization interface in mlir.
-  The new mlir bufferization interface is required by jax 0.4.29 or higher.
+* The bufferization of custom Catalyst dialects has been migrated to the new one-shot bufferization 
+  interface in MLIR. The new MLIR bufferization interface is required by JAX v0.4.29 or higher.
   [(#1027)](https://github.com/PennyLaneAI/catalyst/pull/1027)
   [(#1686)](https://github.com/PennyLaneAI/catalyst/pull/1686)
   [(#1708)](https://github.com/PennyLaneAI/catalyst/pull/1708)
   [(#1740)](https://github.com/PennyLaneAI/catalyst/pull/1740)
   [(#1751)](https://github.com/PennyLaneAI/catalyst/pull/1751)
   [(#1769)](https://github.com/PennyLaneAI/catalyst/pull/1769)
 
-* Redundant `OptionalAttr` is removed from `adjoint` argument in `QuantumOps.td` TableGen file
+* The package name of the Catalyst distribution has been updated to be consistent with
```

Comment:
Hmm I don't think this is internal. It is very public in the sense that official package name has changed. However, it shouldn't affect users in most common interactions.